code/game/objects/items/rogueweapons/melee/special.dm

```diff
@@ -132,17 +142,7 @@
 
 /obj/item/rogueweapon/mace/stunmace/funny_attack_effects(mob/living/target, mob/living/user, nodmg)
 	. = ..()
-	if(on)
-		target.electrocute_act(5, src)
-		charge -= 33
-		if(charge <= 0)
-			on = FALSE
-			charge = 0
-			update_icon()
-			if(user.a_intent)
-				var/datum/intent/I = user.a_intent
-				if(istype(I))
-					I.afterchange()
```

Comment:
wait so stun maces lost their stun entirely for more damage?

Reply:
they have the same values of cudgels now. Its a temporary measure while I think of less overpowered stun effects

Reply:
its just cuz I know noble people like to use these to troll ingame